src/indexer/index_writer.rs

```diff
@@ -461,6 +461,15 @@ impl IndexWriter {
         self.segment_updater.garbage_collect_files()
     }
 
+    /// Clear the index
+    /// Enables users to rebuild the index,
+    /// by clearing and resubmitting necessary documents
+    /// Garbage collects unused files after removing segments
+    pub fn clear(&mut self) -> Result<()> {
```

Comment:
Return an `opstamp` associated to this operation.

Reply:
Will do. I want to clarify something, before I spend any time writing regexes and sending a chunky commit that touches many files. 

I was thinking of type aliasing u64 as TStamp or OpStamp
```rust
pub type Opstamp = u64;
```

I think it will improve readability instead of seeing methods like 
https://github.com/tantivy-search/tantivy/blob/28323bbb4395278da46e80c3bf1e0fae4783eeef/src/indexer/index_writer.rs#L629-L634

to get 

```rust
pub fn delete_term(&mut self, term: Term) -> Opstamp 
```

I know IDEs and lookups can help, but I thought aliasing is an easy readability improvement that I can make in this commit.

Thoughts?

Reply:
I'm ok with introducing a type alias for Opstamp